cmds/tui.c

```diff
@@ -2578,6 +2735,10 @@ static void tui_main_loop(struct opts *opts, struct uftrace_data *handle)
 		case 'v':
 			tui_debug = !tui_debug;
 			break;
+		case 'f':
+			tui_window_graph_field(graph_field_flag, &selected_field);
```

Comment:
Do you think we need to keep the selected_field here?  Also graph_field_flag can be initialized inside the function so we can make it not take any arguments.

Reply:
I agree with initializing `graph_field_flag` inside `tui_window_graph_field` function.

But the reason I kept `selected_field` in `tui_main_loop` function is to remember the location a user last moved.
How do you think about it?

Reply:
Well.. it might be helpful but I think we can simply start with the first item always.

Reply:
I see. I'll change it!